lalrpop/src/rust/mod.rs

```diff
@@ -178,6 +178,7 @@ impl<W:Write> RustWrite<W> {
     }
 
     pub fn write_module_attributes(&mut self, grammar: &Grammar) -> io::Result<()> {
+        rust!(self, "#![cfg_attr(rustfmt, rustfmt_skip)]");
```

Comment:
Hmm, it feels a bit odd to have this here... but reading the code I'm not sure where else it would go. There's always this tension about rust/mod.rs being sort of "general purpose" but also kinda' tailored to the job it's asked to do. Still, I might rather have this in the `emit_module_attributes` in `build/mod.rs` versus here.

Reply:
Most of my time working on this was spent trying to find a place that would output the attribute in the correct location. :)

So, I don't really know the code base well enough to have a deeply informed opinion. There are two reasons to keep it here:

1. It's already tested. (It probably doesn't mean much given how much testing I did, but people often ignore this factor.)
2. I think every code generator for rust (assuming the rust module isn't for helping format rust code) would want to ignore rustfmt. That's not specific to lalrpop in any way.

Reply:
OK, I don't feel that strongly either way.